bliss/inference.py

```diff
@@ -18,29 +18,33 @@
 
 
 def reconstruct_scene_at_coordinates(
+    n_samples: Optional[int],
```

Comment:
isn't it a bit strange to have theh samples be optional as a positional argument rather than a keyword argument?

Reply:
Yeah, I was thinking the user should have to explicitly pass "None" to know that they are getting the MAP, vs passing a 1 to get a single sample. It might just be better to split up the methods as you point out earlier